keras/legacy/interfaces.py

```diff
@@ -74,3 +74,50 @@ def wrapper(*args, **kwargs):
 
         return func(*args, **kwargs)
     return wrapper
+
+def legacy_embedding_support(func):
+    """Function wrapper to convert the `Embedding` constructor from Keras 1 to 2.
+
+    # Arguments
+        func: `__init__` method of `Embedding`.
+
+    # Returns
+        A constructor conversion wrapper.
+    """
+    @six.wraps(func)
+    def wrapper(*args, **kwargs):
+        if len(args) > 1:
+            # The first entry in `args` is `self`.
+            raise TypeError('The `Embedding` layer cannot have any positional argument')
```

Comment:
`args` is the list of positional arguments passed by the user. It could have any length.